duckduckgo/iOS
# home row CTA experiment 2

Task/Issue URL: https://app.asana.com/0/392891325557410/716678893422661
Tech Design URL:
CC:

**Description**:

Some refactoring to accommodate a slightly better structure in order to be able to run experiment 1 and 2 side by side using variants for selection.

**Steps to test this PR**:

Variant Assignment
1. Fresh install a number of times and ensure variant is being allocated to `me` and `mf` (should be 50% each)

Default CTA (`me`)
1. Edit StatisticsUserDefaults to return `me` for the stored variant
1. App should show experiment 1 CTA which behaves as before (if tapped is dismissed)

Experiment 2 CTA (`mf`)
1. Edit StatisticsUserDefaults to return `mf` for the stored variant
1. App should show experiment 2 CTA 
1. Animates in from the bottom when first shown
1. Launches "home row" instructions when 'show me' is tapped but does not get dismissed
1. Animates out when dismissed using 'no thanks' button
1. Once dismissed does not re-reappear
1. Supports Portrait (fully width) and Landscape (restricted width)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
